mobile-app/app/tailwind.config.js

```diff
@@ -108,6 +116,10 @@ module.exports = {
           500: '#FF9F0A',
           600: '#FFBC54',
           700: '#BF7708'
+        },
+        darkblue: {
```

Comment:
You can remove this, it will be added with Harsh's PR - https://github.com/DeFiCh/wallet/blob/b7db18cc5fb064293e6e9d0a0708d977d9d1095b/mobile-app/app/tailwind.config.js#L123

Reply:
Updated